### PR TITLE
repl: Add a little progress bar and a commit indicator

### PR DESCRIPTION
**Pull request by alimpfard**
Just adds a little commit hash and a basic loading progress bar so I don't have to stare at the void while the wasm file is downloaded.
I _could_ provide a download progress, but that's a pain, so...

(Will have to merge before https://github.com/SerenityOS/serenity/pull/16133)

Here's a [preview](https://remote.cxbyte.me/libjs-repl/repl/).

### CODE REVIEW

File: repl/libjs.js [hunk 1]
```diff
@@ -4272,8 +4272,8 @@ var dynCall_ji = Module["dynCall_ji"] = createExportWrapper("dynCall_ji");
 /** @type {function(...*):?} */
 var dynCall_jiji = Module["dynCall_jiji"] = createExportWrapper("dynCall_jiji");
 
-var ___start_em_js = Module['___start_em_js'] = 10075532;
-var ___stop_em_js = Module['___stop_em_js'] = 10075623;
+var ___start_em_js = Module['___start_em_js'] = 10076332;
+var ___stop_em_js = Module['___stop_em_js'] = 10076423;
```
Comment (linusg) [inline repl/libjs.js:4276]: should this and the wasm regen be in a separate commit?

File: repl/main.js [hunk 2]
```diff
@@ -6,17 +6,47 @@ const outputTemplate = document.getElementById("repl-output-template");
...
+      loadingProgress.max = total;
+      loadingProgress.value = loaded;
+    } else {
+      delete loadingProgress.max;
+      delete loadingProgress.value;
+    }
+  }
+
+  updateLoading("Build Information", { known: false });
+  fetch("/repl/commit")
```
Comment (linusg) [inline repl/main.js:28]: I think ideally we'd integrate this in libjs.js - I assume you're not manually copying a git commit into the file every time, so perhaps we could simply append something like `Module.SERENITYOS_COMMIT = "xxx";` to the file when it's being generated?

**Review by linusg — COMMENTED** [review]
Nice domain you got there, but it doesn't seem to load here 👀

File: repl/main.js (hunk 2 above)
Comment (alimpfard) [reply repl/main.js:28]: I could, the build job just echoes it to that `commit` file atm

**Comment by alimpfard** [comment]
>doesn't seem to load here

I forgot the main domain was in Iran...oops

File: repl/libjs.js (hunk 1 above)
Comment (alimpfard) [reply repl/libjs.js:4276]: ah, we don't actually need the libjs.* changes, I'll remove them from this PR.

File: repl/main.js (hunk 2 above)
Comment (alimpfard) [reply repl/main.js:28]: over here: [SerenityOS/serenity@`dac1685` (#16133)](https://github.com/SerenityOS/serenity/pull/16133/commits/dac1685e5c3517fcd0c9924b97026db6c6f1789e#diff-7985cab97a05bfe5fc9e7b10f0a11e6889e2eddf3f937bb16f85e14409e3b35eR89)

Comment (alimpfard) [reply repl/main.js:28]: I'll just make it append the commit hash into `libjs.js`, that seems like the best option.
